packages/sdks-tests/src/tests/css-properties.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from '@playwright/test';
+import { excludeTestFor, test } from './helpers/index.js';
+
+test.describe('CSS Properties from Builder Content', () => {
```

Comment:
could we have this same test rerun once like this, and once with Javascript disabled? that way we can test the styles coming from SSR.

Reply:
yeah that would be awesome! thanks for this suggestion! Should only ssr frameworks run those tests (with js disabled) because client side frameworks will still need js right?

Reply:
correct!